src/widgets/checkboxes/src/components/CheckboxesWidget.js

```diff
@@ -139,14 +142,18 @@ class CheckboxesWidget extends Component<WidgetProps> {
       withNone = false,
       withOther = false,
       noneText,
+      otherText = 'Other'
```

Comment:
Unless @solomontang objects, let's do this for `noneText` as well.

Reply:
I thought we already did this. Maybe it was just for the radio widget? But sounds good